receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go

```diff
@@ -77,8 +77,8 @@ func (s *scraper) scrape(_ context.Context) (pdata.Metrics, error) {
 	if s.config.CPUAverage {
 		divisor := float64(runtime.NumCPU())
 		avgLoadValues.Load1 = avgLoadValues.Load1 / divisor
-		avgLoadValues.Load5 = avgLoadValues.Load1 / divisor
-		avgLoadValues.Load15 = avgLoadValues.Load1 / divisor
+		avgLoadValues.Load5 = avgLoadValues.Load5 / divisor
```

Comment:
Can you add a unit test to verify this change / regressions?

Reply:
sure

Reply:
@dmitryax  I am seeing that Test case is aleady [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_test.go#L119)
If there is any specific thing you want me to check, please let me know.

Reply:
Respectfully, there is a unit test: it verifies the data is present.

Leaving the code wrong to berate the contributor for not working out how to make a test that calculates a 15 minute average seems like an unnecessary harm.